assume/common/outputs.py

```diff
@@ -380,6 +384,8 @@ async def on_stop(self):
 
             dfs.append(df)
 
+        # remove all empty dataframes
+        dfs = [df for df in dfs if not df.empty]
```

Comment:
why should there be an empty dataframe? are you sure this still happens?

Reply:
Idk why, but I was getting this FutureWarning again. So I added this just in case to remove the warning